phpIPAM/phpIPAM.py

```diff
@@ -113,7 +113,7 @@ def ticket_extend(self):
 
     def authorization(self, controller):
         "Check the authorization of a controller and get a list of methods"
-        return self.__query("/%s/" %(controller))['methods']
```

Comment:
I think methods is correct here. What needs to be changed is the Method needs to be 'OPTIONS" not "GET"